collectors/apps.plugin/apps_groups.conf

```diff
@@ -159,6 +159,12 @@ logs: ulogd* syslog* rsyslog* logrotate systemd-journald rotatelogs
 nms: snmpd vnstatd smokeping zabbix* monit munin* mon openhpid watchdog tailon nrpe
 splunk: splunkd
 azure: mdsd *waagent* *omiserver* *omiagent* hv_kvp_daemon hv_vss_daemon *auoms* *omsagent*
+datadog: *datadog*
+edgedelta: edgedelta
+newrelic: newrelic*
+google-agent: *google_guest_agent* *google_osconfig_agent*
+nvidia-smi: nvidia-smi
+htop: htop
```

Comment:
Why do you think we need `htop`? It is not a daemon and I think it consumes very few CPU/mem.

Reply:
I had the same doubt, I am waiting for the answer to approve the PR.

Reply:
its useful for signaling when you or someone else was working on the system and so using htop locally.

Reply:
This doesn't really answer my question, my doubts:

- not a daemon, but just a tool (I think people use it for several seconds to check something quickly).
- uses little CPU/mem => and doesn't contribute a lot to overall CPU usage => it is ok to have it in "other".

So I don't see how having `htop` as a dimension is any help. My opinion is that the majority of users don't care if somebody using "using htop locally".